### PR TITLE
mamba, mamba-repoquery: add pages

### LINKED ISSUE #8488: page request: mamba
### Command description

Mamba is a fast drop-in replacement for the conda environment manager

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [X] macOS
- [ ] sunOS
- [X] Windows

### PR DESCRIPTION
**Pull request by BSGalvan**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): `mamba-0.27.0`**

Should close #8488.

### CODE REVIEW

**Comment by kbdharun** [comment]
@corneliusroemer can you check this PR?

> Thanks in advance

File: pages/common/mamba-repoquery.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Search for all packages satisfying constraints on this query:
+
+`mamba repoquery search {{sphinx<5}}`
+
+- List the dependencies of a package installed in the currently activated environment, in a tree format:
+
+`mamba repoquery depends --tree {{scipy}}`
+
+- Look for other packages who need a particular package installed in the currently activated environment (i.e., inverse of `depends`):
```
Comment (corneliusroemer) [inline pages/common/mamba-repoquery.md:18]:
```suggestion
- Print packages in the current environment the require a particular package to be installed (i.e. inverse of `depends`):
```

**Review by corneliusroemer — CHANGES_REQUESTED** [review]
Really good start, just a few changes would make it even better

File: pages/common/mamba.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Activate an environment:
+
+`mamba activate {{environment_name}}`
+
+- List all installed packages in the currently activated environment:
+
+`mamba list`
+
+- Install packages into the current environment, specifying the package [c]hannel:
```
Comment (corneliusroemer) [inline pages/common/mamba.md:35]: Please move install up to second position after create, it's very often used

File: pages/common/mamba.md [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
 
 `mamba create --name {{environment_name}} {{python=3.10 matplotlib}}`
 
-- List all environments:
+- Update all packages in environment:
 
-`mamba info --envs`
+`mamba update --all`
 
-- Activate an environment:
```
Comment (corneliusroemer) [inline pages/common/mamba.md]: I think we could add deactivate back if there's not already 8 commands, sorry for the confusion

**Review by corneliusroemer — CHANGES_REQUESTED** [review]
Almost great :)

**Comment by kbdharun** [comment]
> Almost great :)

Thank you so much for the help.

File: pages/common/mamba.md (hunk 3 above)
Comment (BSGalvan) [reply pages/common/mamba.md]: There are already 8 commands now, so leave as is?

File: pages/common/mamba.md (hunk 2 above)
Comment (BSGalvan) [reply pages/common/mamba.md]: Done!

**Review by corneliusroemer — APPROVED** [review]

**Comment by corneliusroemer** [comment]
Approved, but my review doesn't count :) @kbdharun

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/mamba-repoquery.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# mamba repoquery
+
+> Efficiently query conda and mamba package repositories and package dependencies.
+> More information: <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#repoquery>.
+
+- Search for all available versions of a particular package:
+
+`mamba repoquery search {{package_name}}`
+
+- Search for all packages satisfying constraints on this query:
```
Comment (marchersimon) [inline pages/common/mamba-repoquery.md:10]: Couldn't we remove this part?
```suggestion
- Search for all packages satisfying specific constraints:
```

File: pages/common/mamba.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Create a new environment, installing the specified packages into it:
+
+`mamba create --name {{environment_name}} {{python=3.10 matplotlib}}`
+
+- Install packages into the current environment, specifying the package [c]hannel:
+
+`mamba install -c {{conda-forge}} {{python=3.6 numpy}}`
+
+- Update all packages in environment:
```
Comment (marchersimon) [inline pages/common/mamba.md:15]:
```suggestion
- Update all packages in the current environment:
```

File: pages/common/mamba.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`mamba update --all`
+
+- Search for a particular package across repositories:
```
Comment (marchersimon) [inline pages/common/mamba.md:19]: Just to keep it uniform with the other pages:
```suggestion
- Search for a specific package across repositories:
```

File: pages/common/mamba.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+`mamba repoquery search {{numpy}}`
+
+- List all environments:
+
+`mamba info --envs`
+
+- Remove unused [p]ackages and [t]arballs from cache:
```
Comment (marchersimon) [inline pages/common/mamba.md:27]:
```suggestion
- Remove unused [p]ackages and [t]arballs from the cache:
```

**Review by marchersimon — APPROVED** [review]
Thanks @BSGalvan

**Comment by BSGalvan** [comment]
Thanks @kbdharun, @corneliusroemer, @marchersimon for the wonderful suggestions and your time :D

**Pull request merged**